src/providers/authProvider/index.ts

```diff
@@ -55,7 +55,12 @@ const authProvider = (dataProvider: DataProvider): AuthProvider => {
           const salt: string = generateSalt()
           const token = encryptData(`${JSON.stringify(clonedUser)}${salt}`)
           setToken(token, salt)
-          await audit(AuditType.LOGIN, 'Logged in')
+          await audit({
+            type: AuditType.LOGIN,
+            activityDetail: 'Logged in',
+            resource: constants.R_USERS,
```

Comment:
Nice thinking, but let's provide `null` for `resource`, please - since we're not affecting the `users` table.